crates/node/src/service/rpc/metrics.rs

```diff
@@ -113,15 +113,15 @@ impl RpcMetrics {
         tracing::trace!(target: "rpc_metrics", "[{transport_label}] on_response started_at={:?}", now);
         tracing::trace!(target: "rpc_metrics::extra", "[{transport_label}] result={}", rp.as_result());
 
-        let micros = now.elapsed().as_micros();
+        let millis = now.elapsed().as_millis();
         tracing::debug!(
             target: "rpc_metrics",
             "[{transport_label}] {} call took {} μs",
```

Comment:
here you should remove μs seconds to mili

Reply:
valid, done!